src/patients/appointments/AppointmentsList.tsx

```diff
@@ -51,10 +46,10 @@ const AppointmentsList = (props: Props) => {
       <br />
       <div className="row">
         <div className="col-md-12">
-          {appointments ? (
-            appointments.length > 0 ? (
+          {data ? (
+            data.length > 0 ? (
               <Table
-                data={appointments}
+                data={data}
```

Comment:
seems like this can be simplified: 

```
if(data === undefined || status === 'loading') { return <Loading />}

if(data.length === 0) { // return warning message) }

// render the component
```